pkg/library/libraryapplyconfiguration/command.go

```diff
@@ -97,6 +105,7 @@ func newApplyConfigurationCommand(applyConfigurationFn ApplyConfigurationFunc, o
 func (f *applyConfigurationFlags) BindFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&f.inputDirectory, "input-dir", f.inputDirectory, "The directory where the resource input is stored.")
 	flags.StringVar(&f.outputDirectory, "output-dir", f.outputDirectory, "The directory where the output is stored.")
+	flags.StringSliceVar(&f.controllersToRun, "controllers-to-run", f.controllersToRun, "An optional list of controller names to run. By default, all controllers are run.")
```

Comment:
the canonical name for this from `kube-controller-manager` is `--controllers`.  Its help indicates 

> A list of controllers to enable. '*' enables all on-by-default controllers, 'foo' enables the controller named 'foo', '-foo' disables the controller named 'foo'

Reply:
renamed to `--controllers`.